src/python_testing/TC_PAVSTI_1_1.py

```diff
@@ -46,13 +48,31 @@
 logger = logging.getLogger(__name__)
 
 
-class TC_PAVSTI_1_1(MatterBaseTest, AVSMTestBase):
+class TC_PAVSTI_1_1(MatterBaseTest, AVSMTestBase, PAVSTIUtils):
     def desc_TC_PAVSTI_1_1(self) -> str:
         return "[TC-PAVSTI-1.1] Verify transmission when trigger type is Manual."
 
     def pics_TC_PAVSTI_1_1(self):
         return ["PAVST.S"]
 
+    @async_test_body
+    async def setup_class(self):
+        th_server_app = self.user_params.get("th_server_app_path", None)
+        if th_server_app:
+            self.server = PushAvServerProcess(server_path=th_server_app)
+        else:
+            self.server = PushAvServerProcess()
+        self.server.start(
+            expected_output="Running on https://0.0.0.0:1234",
```

Comment:
is the server running on localhost, which is http://127.0.0.1?

Reply:
The server is spun on wildcard so that the server can be accessible when running TC on Test Harness from the browser UI. The camera app currently still uses localhost to connect to the server. The hostname and url with respect to TLS will be updated once camera-app supports it.

Reply:
Thank you for your review. The server is spun on wildcard so that the server can be accessible when running TC on Test Harness from the browser UI. The camera app currently still uses localhost to connect to the server. The hostname and url with respect to TLS will be updated once camera-app supports it.